fn-fdk.js

```diff
@@ -223,6 +223,20 @@ function getInputHandler (inputMode) {
   }
 }
 
+function logFramer (ctx) {
+  let framer = process.env['FN_LOGFRAME_NAME']
+  if (framer !== '') {
+    let valueSrc = process.env['FN_LOGFRAME_HDR']
```

Comment:
If I provide `FN_LOGRAME_NAME` but not `FN_LOGFRAME_HDR` this will silently do nothing. This seems like unintuitive behaviour to me. Is this consistent with the other FDKs?

Reply:
I think this is fine. FDK should be defensive and ignore faulty agent/runner.